src/main/java/ch/securify/analysis/AbstractDataflow.java

```diff
@@ -75,7 +75,35 @@
     protected String SOUFFLE_RULES;
     protected final String TIMEOUT_COMMAND = System.getProperty("os.name").toLowerCase().startsWith("mac") ? "gtimeout" : "timeout";
 
+    protected void checkSouffleInstallation() {
```

Comment:
add `throws` or change this to be a boolean check instead of throwing errors (which I would prefer, perhaps rename it to `isSouffleInstalled` or something like that).